qa/rspec/commands.rb

```diff
@@ -135,6 +135,11 @@ def install(options = {})
       client.install(package)
     end
 
+    def write_default_pipeline()
+      # defines a minimal pipeline so that the service is able to start
+      client.write_pipeline(pipeline_string="input { heartbeat {} } output { null {} }")
```

Comment:
The current method works since `pipeline_string="input { heartbeat {} } output { null {} }"` just evals to an assignment that returns the right hand side:
```ruby
> pipeline_string="input { heartbeat {} } output { null {} }"
 => "input { heartbeat {} } output { null {} }" 
```
So in fact the code could use a different variable name and still work:

```ruby
      client.write_pipeline(potatoes="input { heartbeat {} } output { null {} }")
```

If one is to use keyword arguments, the `write_pipeline` method would have to be rewritten as :

```ruby
    def write_pipeline(pipeline_string: )
      run_command("bash -c \"echo '#{pipeline_string}' >/etc/logstash/conf.d/pipeline.conf\"")
    end
```

Then invoked as such:
```ruby
      client.write_pipeline(pipeline_string: "input { heartbeat {} } output { null {} }")
```

Otherwise, to keep it simple, we can leave the method without keyword args and just use the first argument:

```suggestion
      client.write_pipeline("input { heartbeat {} } output { null {} }")
```

Reply:
Oops my Python background exposure was bound to get me trip on something so obvious but different in Ruby!